applications/nrf5340_audio/src/bluetooth/le_audio_bis_gateway.c

```diff
@@ -484,6 +484,8 @@ int le_audio_enable(le_audio_receive_cb recv_cb)
 
 	ARG_UNUSED(recv_cb);
 
+	LOG_INF("Starting broadcast gateway %s", CONFIG_BT_AUDIO_BROADCAST_NAME);
```

Comment:
Both the changes in streamctrl.c and le_audio_bis_gateway.c looks identical to the changes in https://github.com/nrfconnect/sdk-nrf/pull/10227

Reply:
Yes as both use button 5 and the logging is used in both. I could have different logging but that doesn't make sense.